contracts/Vault.sol

```diff
@@ -360,22 +360,38 @@ contract Vault is
         return maxInvestableAssets - alreadyInvested;
     }
 
+    function investState()
+        public
+        view
+        returns (uint256 maxInvestableAmount, uint256 alreadyInvested)
+    {
+        maxInvestableAmount = totalUnderlying().percOf(investPerc);
+        alreadyInvested = strategy.investedAssets();
+    }
+
     /// See {IVault}
-    function updateInvested(bytes calldata data)
+    function updateInvested(bool invest, bytes calldata data)
```

Comment:
I'm still confused :s so if the backend sends a `false`, what happens?

Reply:
I see. But do you think that's useful? What if we just do whatever it's supposed to do?

Reply:
I think this is useful, because it requires investable amount check in backend.
If we don't have this, the backend can call this func any time - even there is very small amount to invest or disinvest.

Reply:
It would be better to make two functions instead of having boolean param,